tensornetwork/tensor.py

```diff
@@ -185,3 +185,6 @@ def __matmul__(self, other: "Tensor") -> "Tensor":
       raise ValueError(errstr)
     array = self.backend.matmul(self.array, other.array)
     return Tensor(array, backend=self.backend)
+
+  def __repr__(self):
+    return self.array.__repr__()
```

Comment:
Could we do something like
`return "tn.Tensor: \" + self.array.__repr__()`?

Reply:
SGTM!